### PR TITLE
improvement(memory_stress): support stress-ng installation on Ubuntu

### PR DESCRIPTION
**Pull request by bentsi**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -2776,9 +2776,13 @@ def disrupt_memory_stress(self):
         no coredump, serious db error occur during the heavy load of memory.
         """
         self._set_current_disruption(f'MemoryStress on {self.target_node}')
-        self.target_node.install_epel()
-        self.target_node.remoter.run('sudo yum install -y stress-ng')
-
+        if self.target_node.distro.is_rhel_like():
+            self.target_node.install_epel()
+            self.target_node.remoter.sudo('yum install -y stress-ng')
+        elif self.target_node.distro.is_ubuntu():
```
Comment (amoskong) [inline sdcm/nemesis.py:2782]: is_ubuntu() or is_debian() ?

Comment (bentsi) [reply sdcm/nemesis.py:2782]: didn't check this on debian so can tell that it works on Ubuntu only, anyway we don't have Debian based Machine Image so this makes supporting Debian currently irrelevant

**Review by yarongilor — APPROVED** [review]
LGTM

File: sdcm/nemesis.py (hunk 1 above)
Comment (amoskong) [reply sdcm/nemesis.py:2782]: You are right.

**Review by amoskong — COMMENTED** [review]
Looks good.

**Pull request merged**